witchcraft/internal/tcpjson/envelope.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2021 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tcpjson
+
+import (
+	"encoding/json"
+	"os"
+)
+
+type SlsEnvelopeV1 struct {
```

Comment:
nit: should be `SLSEnvelopeV1`.

Is this specification taken from somewhere? If so, might be helpful to include link.

Reply:
This is mirrored from the internal sls-spec, which has not been migrated to witchcraft-api yet.
I updated this to `LogEnvelopeV1` instead of SLS since we're re-defining it internally.